cli.mjs

```diff
@@ -152,6 +249,10 @@ async function actionPrompt() {
         name: "Download data",
         value: "download",
       },
+      {
+        name: "Register new participant under study",
```

Comment:
Can they register to new participants AND new studies or just new participants on an existing study?

Reply:
It should automatically register the new study if the study is not found! I will update this description to make it more clear.